engine/dlib/src/dlib/log.cpp

```diff
@@ -323,6 +351,7 @@ void dmLogInitialize(const dmLogParams* params)
     thread = dmThread::New(dmLogThread, 0x80000, 0, "log");
     g_dmLogServer->m_Thread = thread;
 
+    g_LogLevel = GetInitLogLevel();
```

Comment:
This is the only real change in this PR, and I don't see a big difference here.

I think you need to explain your use case in further detail.